simcore/simulation/library/filament.cpp

```diff
@@ -242,6 +243,12 @@ void Filament::InsertAt(double *pos, double *u) {
   Logger::Trace("Inserting filament at [%2.1f, %2.1f, %2.1f] with orientation"
                 "[%2.1f, %2.1f, %2.1f]",
                 pos[0], pos[1], pos[2], u[0], u[1], u[2]);
+  if(custom_set_tail_==1){
+    for (int i=0; i<n_dim_; ++i){
+    pos[i]+=u[i]*length_*.5;   
+    Logger::Trace("Insert file locations are the locations of tails");
```

Comment:
Formatting: indent here. Use a linter/fixer to make this automatic and painless.